Assets/MixedRealityToolkit.Services/SceneSystem/MixedRealitySceneSystem.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.MixedReality.Toolkit.SceneSystem
     /// This part handles the runtime parts of the service.
     /// </summary>
     [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/SceneSystem/SceneSystemGettingStarted.html")]
-    public partial class MixedRealitySceneSystem : BaseCoreSystem, IMixedRealitySceneSystem
+    public partial class MixedRealitySceneSystem : BaseCoreSystem
```

Comment:
Shouldn't this stay the same?

```suggestion
    public partial class MixedRealitySceneSystem : BaseCoreSystem, IMixedRealitySceneSystem
```

Reply:
Unneeded. BaseCoreSystem already specifies IMixedRealityService

Reply:
Good catch! My bad reading.. :D